src/h5web/visualizations/containers/NxSpectrumContainer.tsx

```diff
@@ -34,6 +35,14 @@ function NxSpectrumContainer(props: VisContainerProps): ReactElement {
 
   const { signal, title, errors, axisMapping } = nxData;
 
+  const { setScaleType } = useLineConfig();
+
+  useEffect(() => {
+    if (signal.scaleType) {
+      setScaleType(signal.scaleType);
+    }
+  }, [setScaleType, signal.scaleType]);
```

Comment:
Tentative implementation

I also consider passing `signal.scaleType` as a `scaleType` prop of `MappedLineVis` and putting the `useEffect` in the `MappedLineVis`.

Let me know what you think.